hack/config-default.sh

```diff
@@ -15,4 +15,3 @@ default_csv_version="${docker_tag/latest/0.0.0}"
 default_csv_version="${default_csv_version/devel/0.0.0}"
 [[ $default_csv_version == v* ]] && default_csv_version="${default_csv_version/v/}"
 csv_version=${CSV_VERSION:-$default_csv_version}
-quay_repository=${QUAY_REPOSITORY:-kubevirt}
```

Comment:
I am late to the party, but why should there be no default repo set? It looks like the templator needs it. How is that related to accessing the internet?

Reply:
When the repo is not set, the templator will skip that step which accesses the Quay repo. So that will solve the offline build issue.
For our releases this isn't a problem, the repo is set in Travis. During development this also isn't a problem, because you should use your own repo anyway.